packages/db/src/resources/contracts.ts

```diff
@@ -33,6 +33,16 @@ export const contracts: Definition<"contracts"> = {
       processedSource: ProcessedSource
       createBytecode: Bytecode
       callBytecode: Bytecode
+      callBytecodeGeneratedSources: [GeneratedSource]
+      createBytecodeGeneratedSources: [GeneratedSource]
+    }
+
+    type GeneratedSource {
+      ast: AST!
+      id: Int!
```

Comment:
Can we call this `index` instead?

Reply:
I don't think this is an index, is it? If so, can you point me to the docs? I wasn't able to find much about this except for what Solidity calls it (which is why I called it `id`) and what data type it is.

Reply:
I can say for a fact that it's a source index; but as for documentation, I don't think there is any.  But when Solidity talks about source IDs, it means source indices.  Same as in the compiler output, you know?  In any case, it's definitely a source index and that's how the debugger uses it.

Reply:
Thanks for the clarification Harry!